frontend/workflows/projectSelector/src/project-selector.tsx

```diff
@@ -171,9 +177,42 @@ const ProjectSelector = () => {
       <StateContext.Provider value={derivedState}>
         <StyledSelectorContainer>
           <StyledWorkflowHeader>
-            {/* TODO: change icon to match design */}
-            <LayersIcon />
+            <LayersOutlinedIcon />
             <StyledWorkflowTitle>Dash</StyledWorkflowTitle>
+            <Tooltip
+              title={
+                <>
+                  <TooltipContainer>
+                    <Typography variant="subtitle3" color="#FFFFFF">
+                      Projects
+                    </Typography>
+                    <Typography variant="body3" color="#E7E7EA">
+                      {projectDescription}
+                    </Typography>
+                  </TooltipContainer>
```

Comment:
Actually I'd probably turn this into a loop

Reply:
turned into a loop